frontend/src/components/EditPsychiatristProfile.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import chevron_down from "@/assets/chevron_down";
+
+const EditPsychiatristProfile = () => {
```

Comment:
Passing in current psychiatrist - use props and pass that in from a parent component.